backends-velox/src/main/scala/org/apache/gluten/backendsapi/velox/VeloxSparkPlanExecApi.scala

```diff
@@ -181,6 +181,11 @@ class VeloxSparkPlanExecApi extends SparkPlanExecApi {
       function: ExpressionTransformer,
       expr: ArrayFilter): ExpressionTransformer = {
     expr.function match {
+      // Transformer for array_compact.
+      case LambdaFunction(function, arguments, _)
+          if function.getClass.getSimpleName.equals("IsNotNull") &&
```

Comment:
Can we simplify it as below?

case LambdaFunction(_: IsNotNull, arguments, _)

Reply:
Yes, already simplified